app/src/main/java/com/amazon/connect/chat/androidchatexample/views/ChatComponents.kt

```diff
@@ -142,13 +143,22 @@ fun SenderChatBubble(message: Message, recentOutgoingMessageID: String? = null)
             }
 
             message.metadata?.status?.let {
-                if (message.id == recentOutgoingMessageID) {
+                val retryEnabled = !arrayOf("Delivered", "Read", "Sending", "Sent").contains(it.status)
```

Comment:
NIT: Since we're re-using this, we should maybe put this in a utils or constants file

Reply:
Sure. This logic is a bit different in attachment as it requires converting the message status type into string. But agreed I can still make this a util